ant-cli/src/commands/analyze.rs

```diff
@@ -104,3 +113,61 @@ fn try_other_types(addr: &str, verbose: bool) {
 
     println!("⚠️ Unrecognized input");
 }
+
+async fn print_closest_nodes(client: &autonomi::Client, addr: &str, verbose: bool) -> Result<()> {
+    use autonomi::PublicKey;
+    use autonomi::chunk::ChunkAddress;
+    use autonomi::graph::GraphEntryAddress;
+
+    macro_rules! println_if_verbose {
+        ($($arg:tt)*) => {
+            if verbose {
+                println!($($arg)*);
+            }
+        };
+    }
+
+    let hex_addr = addr.trim_start_matches("0x");
+
+    println_if_verbose!("Querying closest peers to address...");
+
+    // Try parsing as ChunkAddress (XorName) first
+    let peers = if let Ok(chunk_addr) = ChunkAddress::from_hex(addr) {
+        println_if_verbose!("Identified as ChunkAddress");
+        client
+            .get_closest_to_address(chunk_addr)
+            .await
+            .map_err(|e| color_eyre::eyre::eyre!("Failed to get closest peers: {e}"))?
+    // Try parsing as PublicKey (could be GraphEntry, Pointer, or Scratchpad)
+    } else if let Ok(public_key) = PublicKey::from_hex(hex_addr) {
+        println_if_verbose!("Identified as PublicKey, using GraphEntryAddress");
+        // Default to GraphEntryAddress for public keys
+        let graph_entry_address = GraphEntryAddress::new(public_key);
+        client
+            .get_closest_to_address(graph_entry_address)
+            .await
+            .map_err(|e| color_eyre::eyre::eyre!("Failed to get closest peers: {e}"))?
+    } else {
+        return Err(color_eyre::eyre::eyre!(
+            "Could not parse address. Expected a hex-encoded ChunkAddress or PublicKey"
+        ));
+    };
+
+    // Sort peers by peer_id for consistent output
+    let mut sorted_peers = peers;
+    sorted_peers.sort_by(|a, b| a.peer_id.cmp(&b.peer_id));
+
+    println!("Found {} closest peers to {}:", sorted_peers.len(), addr);
+    for (i, peer) in sorted_peers.iter().enumerate() {
+        println!("{}. Peer ID: {}", i + 1, peer.peer_id);
```

Comment:
prefer the block of code to be : 
```
if verbose {
    println!("{}. Peer ID: {} - {:?}", i + 1, peer.peer_id, peer.addrs);
} else {
    println!("{}. Peer ID: {}", i + 1, peer.peer_id);
}
```

Reply:
As addresses are very long this will make the printing very ugly and hard to read.